src/main/sensors/acceleration.c

```diff
@@ -497,28 +498,25 @@ void accUpdate(timeUs_t currentTimeUs, rollAndPitchTrims_t *rollAndPitchTrims)
 
     applyAccelerationTrims(accelerationTrims);
 
-    const timeDelta_t sampleDeltaUs = currentTimeUs - lastTimeSampledUs;
-    lastTimeSampledUs = currentTimeUs;
-    accumulatedTimeUs += sampleDeltaUs;
+    ++accumulatedMeasurementCount;
     for (int axis = 0; axis < XYZ_AXIS_COUNT; axis++) {
-        accumulator[axis] += acc.accSmooth[axis] * sampleDeltaUs;
+        accumulatedMeasurements[axis] += acc.accSmooth[axis];
     }
 }
 
-bool accGetAccumulationAverage(float *accumulation)
+bool accGetAccumulationAverage(float *accumulationAverage)
 {
-    if (accumulatedTimeUs > 0) {
-        const float accumulatedTimeS = accumulatedTimeUs * 1e-6;
```

Comment:
(during accumulation, value was multiplied by us, then divided by s, resulting in 1e6 scaling ... )